process/graduation-proposal-template.md

```diff
@@ -0,0 +1,23 @@
+# [Project] Graduation Proposal
+
+_**Insert introduction. See previous proposals for examples. This section should address from a broad perspective why the project feels they are ready to graduation and can state any major accomplishments or milestones.**_
+
+## Graduation State Criteria
+_**Project should address each graduation criteria listed below**_
+
+### * Have committers from at least two organizations.
+
+### * Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge/(https://bestpractices.coreinfrastructure.org/).
+
+### * Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
+
+### * Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+
+### * Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
+
+## Incubation Details
+_**Project should address each area listed below**_
+
+### * Link to Incubation Due Dilligence(DD) Document
```

Comment:
Does a project already in incubation need to do the incubation DD prior to going for graduation? This will apply to projects currently in the incubation state.

Reply:
Projects currently in incubation should already have an incubation DD doc. Before SIGs, TOC members facilitated DD. That document can be referenced and used here.

Reply:
I can't find any reference to this doc. I looked for TUF, Jaeger, and gRPC. (I'm assuming k8s did not have one) Is there a standard place where these go?

Reply:
sp: `diligence` not `dilligence`